site/content/in-dev/unreleased/metastores.md

```diff
@@ -23,33 +23,51 @@ type: docs
-Configure the `metaStoreManager` section in the Polaris configuration (`polaris-server.yml` by default) as follows:
-```
-metaStoreManager:
-  type: eclipse-link
-  conf-file: META-INF/persistence.xml
-  persistence-unit: polaris
-```
 
-`conf-file` must point to an [EclipseLink configuration file](https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm)
+Configure the `polaris.persistence` section in your Polaris configuration file
+(`application.properties`) as follows:
 
-By default, `conf-file` points to the embedded resource file `META-INF/persistence.xml` in the `polaris-eclipselink` module.
+```
+polaris.persistence.type=eclipse-link
+polaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml
+polaris.persistence.eclipselink.persistence-unit=polaris
+```
 
-In order to specify a configuration file outside the classpath, follow these steps.
-1) Place `persistence.xml` into a jar file: `jar cvf /tmp/conf.jar persistence.xml`
-2) Use `conf-file: /tmp/conf.jar!/persistence.xml`
+Alternatively, configuration can also be done with environment variables or system properties. Refer
+to the [Quarkus Configuration Reference] for more information.
 
-## EclipseLink Configuration - persistence.xml
-The configuration file `persistence.xml` is used to set up the database connection properties, which can differ depending on the type of database and its configuration.
+The `configuration-file` option must point to an [EclipseLink configuration file]. This file, named
+`persistence.xml`, is used to set up the database connection properties, which can differ depending
+on the type of database and its configuration.
 
-Check out the default [persistence.xml](https://github.com/apache/polaris/blob/main/extension/persistence/eclipselink/src/main/resources/META-INF/persistence.xml) for a complete sample for connecting to the file-based H2 database. 
+[Quarkus Configuration Reference]: https://quarkus.io/guides/config-reference
+[EclipseLink configuration file]: https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm
 
 Polaris creates and connects to a separate database for each realm. Specifically, the `{realm}` placeholder in `jakarta.persistence.jdbc.url` is substituted with the actual realm name, allowing the Polaris server to connect to different databases based on the realm.
 
 > Note: some database systems such as Postgres don't create databases automatically. Database admins need to create them manually before running Polaris server.
+
+A single `persistence.xml` can describe multiple [persistence units](https://eclipse.dev/eclipselink/documentation/2.6/concepts/app_dev001.htm). For example, with both a `polaris-dev` and `polaris` persistence unit defined, you could use a single `persistence.xml` to easily switch between development and production databases. Use the `persistence-unit` option in the Polaris server configuration to easily switch between persistence units.
```

Comment:
It'd be nice to give an example of a single file with multiple units. But it isn't a blocker to me. We can improve on it later.